CrowdStrike/ember-toucan-core
# Add lock icon for disabled and readonly states

## 🚀 Description

This PR adds the lock icon for disabled and readonly states according to our designs.

---

## 🔬 How to Test

- Visit https://844e929c.ember-toucan-core.pages.dev/docs/components/checkbox-group-field and the following components
  - checkbox-field
  - checkbox-group-field
  - file-input-field
  - input-field
  - radio-group-field
  - textarea-field
- Scroll down to the "All UI States" section(s)
- Verify the lock icon appears for both disabled and readonly states

---

## 📸 Images/Videos of Functionality

### Iconography

While I was in here, I adjusted some of the iconography in our demos to be a bit closer to what we'd prefer.

| BEFORE  | AFTER |
| ------------- | ------------- |
| <img width="416" alt="Screenshot 2023-06-20 at 7 56 19 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/269b0285-450d-4486-89f1-48192992a23f">  | <img width="400" alt="Screenshot 2023-06-20 at 7 56 39 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/7024f6a9-4a53-4ccd-a539-d56607478fdd">  |





### Readonly + Disabled Locks

A few before and after pictures of the readonly and disabled states.

| BEFORE  | AFTER |
| ------------- | ------------- |
| <img width="370" alt="Screenshot 2023-06-20 at 7 52 01 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/6cb14c3c-1855-4e9c-9bf9-ca248d68343c">  | <img width="376" alt="Screenshot 2023-06-20 at 7 51 30 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/003c15b3-a63c-4f03-aa6c-725773f9a3bc">  |
| <img width="408" alt="Screenshot 2023-06-20 at 7 52 45 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/1599c92e-3f95-40ff-a54e-b76e68ccb3d7">  | <img width="410" alt="Screenshot 2023-06-20 at 7 53 03 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/d5b6b318-1e1a-4265-b3c6-1e027234ec7c">  |
| <img width="543" alt="Screenshot 2023-06-20 at 7 55 04 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/ed40fde1-bdc9-4551-97ed-509228800c54">  | <img width="537" alt="Screenshot 2023-06-20 at 7 54 54 PM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/acb62271-4931-4b27-8e13-ba80f9ac10df"> |
